pkg/handlers/healthProbes.go

```diff
@@ -7,17 +7,39 @@ import (
 	"net/http"
 
 	"github.com/golang/glog"
+	"github.com/open-cluster-management/insights-client/pkg/monitor"
+	"github.com/open-cluster-management/insights-client/pkg/retriever"
 )
 
 // LivenessProbe is used to check if this service is alive.
 func LivenessProbe(w http.ResponseWriter, r *http.Request) {
-	glog.V(2).Info("livenessProbe")
+	glog.V(2).Info("livenessProbe - Checking local cluster id.")
+	monitor := monitor.NewClusterMonitor()
+
+	//Get local-cluster id , if -1 is returned then service will
+	// not be able to get to cloud.redhat.com
+	if monitor.GetLocalCluster() == "-1" {
+		// Respond with error.
+		glog.Warning("Cannot get local-cluster id.")
+		http.Error(w, "Cannot get local-cluster id.", 503)
```

Comment:
Maybe, add `Cannot reach CRC` to the error message to make it clearer what the error means.